### PR TITLE
REF: make testkraken pip-installable

### PR DESCRIPTION
**Pull request by kaczmarj**
This PR:

- restructures the code as a package, under `testkraken`
- adds `versioneer` to handle versioning
- adds a skeleton of a command-line interface. this will be updated in subsequent PR(s)
- updates the README to use the current YAML spec

### CODE REVIEW

File: testkraken/cli.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"""Command-line interface for Testkraken."""
+
+import click
+
+@click.command(help="Run the workflow in PATH.")
+@click.argument('path', type=click.Path(exists=True))
+def main(path):
+    print("Not implemented yet. In the future, will run from {}".format(path))
+
+# QUESTION(kaczmarj): what commands should be available?
```
Comment (djarecka) [inline testkraken/cli.py]: `testrunner.runner` for now

File: README.md [hunk 2]
```diff
@@ -1,49 +1,42 @@
...
+  - [reg:rel_error:white, reg:rel_error:gray]
+  - [reg:rel_error:Right-Hippocampus, reg:rel_error:Right-Amygdala]
+- function: barplot_all_rel_error
+  var_list: []
+script: run_demo_workflow.py
+tests:
+- file: [output/AnnArbor_sub16960/segstats.json, output/AnnArbor_sub20317/segstats.json,
+    output/AnnArbor_sub38614/segstats.json]
+  name: reg
+  script: check_output.py
```
Comment (djarecka) [inline README.md]: thank you for your updates! I will review this after I add some small changes I have in mind

**Comment by djarecka** [comment]
@kaczmarj - could you please add installation part to the README

**Comment by kaczmarj** [comment]
@djarecka - not at the moment, i want to work on the command-line interface in a separate PR.

**Pull request merged**